modules/test/playwright/tests/portal-workflow-kaleo-designer-web/notification.spec.ts

```diff
@@ -193,3 +210,64 @@ test('create a notification using role type', async ({
 		roleTypeNotification
 	);
 });
+
+test('notification receptionType on source remains the same after clicking on notification link on diagram', async ({
+	apiHelpers,
+	diagramViewPage,
+	page,
+	processBuilderPage,
+}) => {
+	const {workflowDefinitionIds} = workflowDefinitions;
```

Comment:
Hey @larissacribeiro, I think we can perform an `array.push` without needed to create this constant